vision/apriltag-py/python/requirements.txt

```diff
@@ -0,0 +1,7 @@
+numpy
```

Comment:
could we combine these with the basestation requirements.txt so the user only has to install requirements once? even if they are not running the vision system